hnn_core/network.py

```diff
@@ -448,20 +450,23 @@ def __init__(self, params, add_drives_from_params=False,
 
     def __repr__(self):
         class_name = self.__class__.__name__
-        s = ("%d x %d Pyramidal cells (L2, L5)"
-             % (self._N_pyr_x, self._N_pyr_y))
-        s += ("\n%d L2 basket cells\n%d L5 basket cells"
-              % (len(self.pos_dict['L2_basket']),
-                 len(self.pos_dict['L5_basket'])))
-        return '<%s | %s>' % (class_name, s)
+        # Dynamically create the description based on the current cell types
+        descriptions = []
```

Comment:
```suggestion
        descriptions = list()
```

just a stylistic suggestion, it's hard to distinguish `[]` and `{}` but `list()` and `dict()` are fairly clear and unmistakable.